Framework/Kernel/inc/MantidKernel/Material.h

```diff
@@ -85,6 +85,7 @@ class MANTID_KERNEL_DLL Material final {
   /// Allow an explicit attenuation profile to be loaded onto the material
   /// that overrides the standard linear absorption coefficient
   void setAttenuationProfile(AttenuationProfile attenuationOverride);
+  void setXRayAttenuationProfile(AttenuationProfile attenuationOverride);
```

Comment:
This is slightly pedantic but I suggest you call the parameter attenuationProfile. In the X-ray case it's not really an override because it's the only source of attenuation information. For neutrons it's a bit different because the .DAT file genuinely is an override to the tabulated neutron cross section information embedded in Mantid